Common/Brokerages/AlphaStreamsBrokerageModel.cs

```diff
@@ -52,15 +52,6 @@ public AlphaStreamsBrokerageModel(AccountType accountType = AccountType.Margin)
         /// <returns>The new slippage model for this brokerage</returns>
         public override ISlippageModel GetSlippageModel(Security security) => new AlphaStreamsSlippageModel();
 
-        /// <summary>
-        /// Force all security types to be restricted to 1.1x leverage
-        ///     - Current restriction to 1.1x is for the AS competition
-        ///     - Will be update in the future
-        /// </summary>
-        /// <param name="security"></param>
-        /// <returns>The leverage for the specified security</returns>
-        public override decimal GetLeverage(Security security) => 1.1m;
```

Comment:
Please copy-paste the base class as an override and change the leverage values to limit the leverage to realistic PB levels; 
https://github.com/QuantConnect/Lean/blob/master/Common/Brokerages/DefaultBrokerageModel.cs#L155

Reply:
Ok. Do we know realistic numbers for PB specific leverage (i.e. guidance from funds)? Otherwise I'll do some research